clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/cache/LoggingInstrumentation.groovy

```diff
@@ -33,7 +33,8 @@ class LoggingInstrumentation implements ExecutionInstrumentation {
 
   @Override
   void executionCompleted(Agent agent) {
-    logger.info("${agent.providerName}:${agent.agentType} completed")
+    def durationMs = Math.round((System.nanoTime() - MetricInstrumentation.getAgentStartTimeNs(agent)) / 1000000)
```

Comment:
I'm not sure there is a solid guarantee that this executionCompletion is going to happen before the MetricInstrumentation executionCompletion (which would have cleared the startTime out of the ThreadLocal)